lib/backend/k8s/resources/customnoderesource.go

```diff
@@ -433,3 +445,11 @@ func (c *customK8sNodeResourceClient) extractResourcesFromAnnotation(node *apiv1
 
 	return kvps, nil
 }
+
+// ExtractResourcesFromNode returns the resources stored in the Node configuration.
+//
+// This convenience method is not guaranteed to be part of the libcalico-go API
```

Comment:
nit: the wording of this seems to imply guarantees that I don't think we necessarily make about other methods in libcalico-go :)

Might be better worded as: "This convenience method is expected to be removed in a future libcalico-go release"